playground/keycloak/realms/weathershop-realm.json

```diff
@@ -0,0 +1,1957 @@
+      }, {
+        "id" : "261d0db4-28c7-4900-a156-01ab4e2483e5",
+        "name" : "view-consent",
+        "description" : "${role_view-consent}",
+        "composite" : false,
+        "clientRole" : true,
+        "containerId" : "65816a45-48d3-4856-b052-c65cb03881d3",
+        "attributes" : { }
+      } ]
+    }
+  },
+  "groups" : [ ],
+  "defaultRole" : {
+    "id" : "f2bd959d-ed9d-4409-af6d-206a4a52cc23",
+    "name" : "default-roles-weathershop",
+    "description" : "${role_default-roles}",
+    "composite" : true,
+    "clientRole" : false,
+    "containerId" : "86683c73-be28-4380-a014-6316c0404192"
+  },
+  "requiredCredentials" : [ "password" ],
+  "otpPolicyType" : "totp",
+  "otpPolicyAlgorithm" : "HmacSHA1",
+  "otpPolicyInitialCounter" : 0,
+  "otpPolicyDigits" : 6,
+  "otpPolicyLookAheadWindow" : 1,
+  "otpPolicyPeriod" : 30,
+  "otpPolicyCodeReusable" : false,
+  "otpSupportedApplications" : [ "totpAppFreeOTPName", "totpAppGoogleName", "totpAppMicrosoftAuthenticatorName" ],
+  "localizationTexts" : { },
+  "webAuthnPolicyRpEntityName" : "keycloak",
+  "webAuthnPolicySignatureAlgorithms" : [ "ES256" ],
+  "webAuthnPolicyRpId" : "",
+  "webAuthnPolicyAttestationConveyancePreference" : "not specified",
+  "webAuthnPolicyAuthenticatorAttachment" : "not specified",
+  "webAuthnPolicyRequireResidentKey" : "not specified",
+  "webAuthnPolicyUserVerificationRequirement" : "not specified",
+  "webAuthnPolicyCreateTimeout" : 0,
+  "webAuthnPolicyAvoidSameAuthenticatorRegister" : false,
+  "webAuthnPolicyAcceptableAaguids" : [ ],
+  "webAuthnPolicyExtraOrigins" : [ ],
+  "webAuthnPolicyPasswordlessRpEntityName" : "keycloak",
+  "webAuthnPolicyPasswordlessSignatureAlgorithms" : [ "ES256" ],
+  "webAuthnPolicyPasswordlessRpId" : "",
+  "webAuthnPolicyPasswordlessAttestationConveyancePreference" : "not specified",
+  "webAuthnPolicyPasswordlessAuthenticatorAttachment" : "not specified",
+  "webAuthnPolicyPasswordlessRequireResidentKey" : "not specified",
+  "webAuthnPolicyPasswordlessUserVerificationRequirement" : "not specified",
+  "webAuthnPolicyPasswordlessCreateTimeout" : 0,
+  "webAuthnPolicyPasswordlessAvoidSameAuthenticatorRegister" : false,
+  "webAuthnPolicyPasswordlessAcceptableAaguids" : [ ],
+  "webAuthnPolicyPasswordlessExtraOrigins" : [ ],
+  "users" : [ {
+    "id" : "8e6aa491-29b4-4701-ac20-a6f0b72ebf2c",
+    "username" : "testuser",
+    "firstName" : "Test",
+    "lastName" : "User",
+    "email" : "testuser@example.com",
+    "emailVerified" : true,
+    "createdTimestamp" : 1719876361930,
+    "enabled" : true,
+    "totp" : false,
+    "credentials" : [ {
+      "id" : "662acb61-35aa-4550-aa3d-1b91d548379b",
+      "type" : "password",
+      "userLabel" : "My password",
+      "createdDate" : 1719876373873,
+      "secretData" : "{\"value\":\"4DfZJfojd4Ef08uoboJ2V+RMpF20yGEbw2pDouefjmcTmAVxE6wPkbLN03u/bAElCm4Y/ndJ9YruH3Q5pFuSLQ==\",\"salt\":\"E8I7GRLxZI2lzpKCI6h/bw==\",\"additionalParameters\":{}}",
```

Comment:
This line is tripping credscan. @eerhardt @joperezr

Reply:
@julioct - is this secret necessary? Can it be removed? I assume/hope it isn't actually a leaked credential that works for something real.

Reply:
That is likely the password of the testuser, prepared for the Playground example. 

We can remove the user from there, but then when running the playground you will first have to register a new user on Keycloak before testing the example.

Would that be OK?

Reply:
You can use environment variables in the imported realm configuration file: [Using Environment Variables within the Realm Configuration Files](https://www.keycloak.org/server/importExport#_using_environment_variables_within_the_realm_configuration_files)

Reply:
@radical @eerhardt I'm removing the test user and associated credentials in [this PR](https://github.com/dotnet/aspire/pull/4985)

Reply:
Thanks @julioct. I've merged that change and will backport to release/8.1 to fix our builds.